packages/@okta/vuepress-theme-prose/components/MobileOnThisPage.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="mobile-on-this-page">
+  <div class="mobile-on-this-page" v-show="options.length > 0">
```

Comment:
I don't understand what this is.  what is `options` and where is it coming from?

Reply:
It's referring to computed property. This is to hide the widget if there are no second level headers. Currently several guide pages don't have second level headers so this widget was showing up without any select options.